cluster/swarm/cluster_test.go

```diff
@@ -187,15 +186,14 @@ func TestLoadImage(t *testing.T) {
 	// create mock client
 	client := mockclient.NewMockClient()
 	apiClient := engineapimock.NewMockClient()
-	client.On("Info").Return(mockInfo, nil)
+	apiClient.On("Info").Return(mockInfo, nil)
+	apiClient.On("ServerVersion").Return(mockVersion, nil)
```

Comment:
For cosmetic reasons mostly. Last time I put it separately, away from all the `client.On(...).Return(...)` calls, but it makes sense to have everything together. No change in functionality.